src/collision_checker.cpp

```diff
@@ -73,16 +73,17 @@
 
 
 CollisionChecker::CollisionChecker(ros::NodeHandle& nh, tf2_ros::Buffer &tf_buffer, 
-		                   ObstaclePoints& op) : tf_buffer(tf_buffer),
-	                                                 ob_points(op)
+                           ObstaclePoints& op) : tf_buffer(tf_buffer),
+                                                     ob_points(op)
 {
     nh.param<std::string>("base_frame", baseFrame, "base_link");
 
     line_pub = ros::Publisher(
                  nh.advertise<visualization_msgs::Marker>("/obstacle_viz", 10));
 
-    max_age = nh.param<float>("max_age", 1.0);
-    no_obstacle_dist = nh.param<float>("no_obstacle_dist", 10.0);
+    max_age = nh.param<float>("max_age", 2.0);
+    no_obstacle_dist_forward = nh.param<float>("no_obstacle_dist_forward", 25.0);
```

Comment:
Because breadcrumb and Magni can in fact see things down to 6cm type of range I feel the proper solution for Polaris is to use launch files to set this 25 and not change the default 10cm everyone else in our customer base has been using.   We must not break legacy for some new platform with unique sensor limitations.

Reply:
> unless it were guarenteed to be only in some branch that only Polaris would use

![Untitled](https://user-images.githubusercontent.com/9977799/110779929-694e1f00-8264-11eb-827a-9e93cd2bc000.png)

I do believe that one's only ever going to be used by the polaris.

> Launch file is the way to solve this

Also this specific cpp file doesn't get launched by ROS, it's included internally by move_basic.cpp and doesn't expose said params outside it. I could do a rework to support that, but I just don't see the point given that it's a polaris specific fix.

Reply:
Ok, so given that I will approve and request that Janez review happen as well as he is familiear with the Polaris usage.  Thanks